bot.js

```diff
@@ -1,57 +1,138 @@
+const defaultLocale = 'en'
+const supportedLocales = ['en', 'ru']
 
-let bot
+let bot = createBot()
 let storage = {}
+i18n.configure({
+  directory: 'data/locales/',
+  locales: supportedLocales,
+  defaultLocale: defaultLocale
+})
+
+function createBot () {
+  if (process.env.NODE_ENV !== 'production') {
+    return new Bot(token, { polling: true })
+  }
 
-if (process.env.NODE_ENV === 'production') {
-  bot = new Bot(token)
+  let bot = new Bot(token)
   bot.setWebHook(process.env.HEROKU_URL + bot.token)
-} else {
-  bot = new Bot(token, { polling: true })
+  return bot
 }
 
-function postNewQuestion (chatID) {
-  const question = quiz.generateQuestion('en')
-  storage[chatID] = { 'answer': question.answer }
+function localeForMsg (msg) {
+  const chatID = msg.chat.id
+  let locale = msg.from.language_code.toLowerCase()
+  if (supportedLocales.includes(locale) === false) {
+    locale = defaultLocale
+  }
+  return (storage[chatID] && storage[chatID].locale) || locale
+}
+
+function postNewQuestion (msg) {
+  const chatID = msg.chat.id
+  const locale = localeForMsg(msg)
+  const question = quiz.generateQuestion(locale)
+  if (!storage[chatID]) {
+    storage[chatID] = {}
+  }
+  storage[chatID].answer = question.answer
 
   const opts = {
-    'reply_markup': { 'keyboard': [question.options], 'one_time_keyboard': true }
+    'reply_markup': {
+      'keyboard': [question.options],
+      'one_time_keyboard': true,
+      parse_mode: 'Markdown' }
   }
   bot.sendMessage(chatID, question.text, opts)
 }
 
 bot.onText(/^[^/]/, function (msg) {
   const chatID = msg.chat.id
+  const locale = localeForMsg(msg)
   if (storage[chatID] == null || storage[chatID].answer == null) {
     bot.sendMessage(
       chatID,
-      'The game has not been started! Start the game by typing /play'
+      i18n.__({
+        phrase: 'wrong_command',
+        locale: locale
+      }),
+      { parse_mode: 'Markdown' }
     )
     return
   }
 
   const answer = storage[chatID].answer
+  let answerReply
   if (msg.text === answer) {
-    bot.sendMessage(chatID, 'Well done!')
+    answerReply = i18n.__({ phrase: 'well_done', locale: locale })
   } else {
-    bot.sendMessage(chatID, 'Oops, the right answer is: ' + answer)
+    answerReply = i18n.__({ phrase: 'wrong_answer_format', locale: locale }, answer)
   }
-
-  postNewQuestion(chatID)
+  bot.sendMessage(chatID,
+    answerReply,
+    { parse_mode: 'Markdown' }).then(() =>
+    postNewQuestion(msg)
+  )
 })
 
 bot.onText(/^\/start$/, (msg) => {
+  const locale = localeForMsg(msg)
+  const chatID = msg.chat.id
+  const welcomeMsg = i18n.__({
+    phrase: 'welcome_format',
+    locale: locale },
+  msg.from.first_name)
   bot.sendMessage(
-    msg.chat.id,
-    `Welcome *${msg.from.first_name}*!`,
+    chatID,
+    welcomeMsg,
     { parse_mode: 'Markdown' }
   )
 })
 
 bot.onText(/^\/play$/, (msg) => {
-  postNewQuestion(msg.chat.id)
+  postNewQuestion(msg)
+})
+
+bot.onText(/\/setLanguage (.+)/, (msg, match) => {
```

Comment:
Users are not programmers :)
```suggestion
bot.onText(/\/language (.+)/, (msg, match) => {
```